src/components/map/createMap.jsx

```diff
@@ -0,0 +1,23 @@
+import { Map, View } from 'ol'
+import TileLayer from 'ol/layer/Tile'
+import OSM from 'ol/source/OSM'
+import React from 'react'
+
+function createMap() {

```

Comment:
Please adhere to using `const` as a convention rather than using `function`, unless hoisting is needed